components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -2429,6 +2429,16 @@ public static boolean isAdaptiveAuthenticationAvailable() {
         return FrameworkServiceDataHolder.getInstance().isAdaptiveAuthenticationAvailable();
     }
 
+    /**
+     * This method is to check whether organization management is enabled.
+     *
+     * @return Organization management feature is enabled or not.
+     */
+    public static boolean isOrganizationManagementEnable() {
```

Comment:
```suggestion
    public static boolean isOrganizationManagementEnabled() {
```

Reply:
fixed in [077830a](https://github.com/wso2/carbon-identity-framework/pull/4165/commits/077830a9e7ab93ed3531984fe2f9c7ce154b8ab2)